test-case/check-audio-equalizer.sh

```diff
@@ -26,16 +27,16 @@ OPT_PARM_lst['d']=1         OPT_VALUE_lst['d']=5
 OPT_OPT_lst['l']='loop'     OPT_DESC_lst['l']='loop count'
 OPT_PARM_lst['l']=1         OPT_VALUE_lst['l']=1
 
-OPT_OPT_lst['s']='sof-logger'   OPT_DESC_lst['s']="Open sof-logger trace the data will store at $LOG_ROOT"
+OPT_OPT_lst['s']='sof-logger'   OPT_DESC_lst['s']="Open sof-logger trace the data will store =at $LOG_ROOT"
 OPT_PARM_lst['s']=0             OPT_VALUE_lst['s']=1
```

Comment:
=at?  is this a mis-change accidently?

Reply:
It was a accident. Will fix this together.